### PR TITLE
Added funder payment reminder button to javascript, model to db, and …

### PR DESCRIPTION
**Pull request by oogetyboogety**
…REST action to Django

##### Description

Adding a button for a contributor to remind a funder to pay out a bounty

##### Checklist

- [X] Read and conforms to the [contributor guidelines](https://docs.gitcoin.co/mk_contributors/)
- [X] Read and conforms to the [style guidelines](https://docs.gitcoin.co/mk_styleguide/)
- [X] Linter status: 100% pass
- [ ] Added relevant tests
- [X] Tests pass and code coverage has not decreased
- [X] Changes don't break existing behavior
- [X] Commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)
- [X] [Code documentation](https://docs.gitcoin.co/mk_contributors/#docstrings)
- [X] Completed: Code review by core team and any requested changes

##### Refers/Fixes

https://github.com/gitcoinco/web/issues/2204

##### Testing and Sign-off

Ran it through my django setup-everything appears to work as it should. 

###### Contributor

- [X] Read and followed the [Contributor Guidelines](https://docs.gitcoin.co/mk_contributors/)
- [X] Tested all changes **locally**
- [X] Verified existing functionality
- [X] Ran `make test` and everything passed!

###### Reviewer

- [ ] Affirm contributor guidelines have been followed and requested changes made
- [ ] CI tests and linting pass
- [ ] No conflicts (migrations, files, etc)
- [ ] Regression tested against staging

###### Funder

- [ ] Validated requested changes were made to specification
- [ ] Bounty payout released to the contributor

### CODE REVIEW

File: app/marketing/management/commands/sync_pull_request_with_bounty_fulfillments.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
+'''
```
Comment (owocki) [inline app/marketing/management/commands/sync_pull_request_with_bounty_fulfillments.py]: should this be scheduled in the crontab? `scripts/crontab`

File: app/marketing/management/commands/sync_pull_request_with_bounty_fulfillments.py [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+from dashboard.models import BountyFulfillment
+from dashboard.utils import record_funder_inaction_on_fulfillment
+from git.utils import get_gh_issue_state, get_interested_actions, post_issue_comment
+from marketing.mails import funder_payout_reminder
+
+
+class Command(BaseCommand):
+
+    help = 'pulls github events associated with settings.GITHUB_REPO_NAME'
```
Comment (owocki) [inline app/marketing/management/commands/sync_pull_request_with_bounty_fulfillments.py]: is this all this mgmt command does?  could use a better desc i think

**Comment by owocki** [comment]
did you test?  i could use a screencap of the email and of a gif of the flow pls

**Comment by owocki** [comment]
merge conflicts need fixing

**Comment by thelostone-mc** [comment]
We've got the dashboard + notifications to take care of this in now !
Thanks @oogetyboogety for the PR :)

**Pull request closed without merging**